pkg/binding/controller.go

```diff
@@ -506,14 +512,14 @@ func (c *Controller) reconcile(ctx context.Context, key util.Key) error {
 			// will add name.
 		}
 
-		c.logger.Info("handled bindingpolicy", "object", util.RefToRuntimeObj(obj))
+		logger.Info("handled bindingpolicy", "object", util.RefToRuntimeObj(obj))
```

Comment:
Logger message should start with a capital letter like the rest of the changed messages.

Reply:
fixed